gravityflow/gravityflowchecklists
# Add Checklist Complete Message Setting with the updated gf-repeater package

## Description
This adds a checklist complete setting under Workflow Settings. We can specify different checklist complete messages for different checklists. By default, the checklist complete message is switched off and can be turned on via the checkbox.

## Testing instructions
- Update your checklist settings to disable and enable the checklist complete message, and note the behavior.
- Change the checklist complete message, and it should be correctly reflecting on a completed checklist.
After testing with Gravity Forms 2.4, test with Gravity Forms 2.5 beta to ensure everything displays correctly there.

## Automated Test Enhancements
N/A

## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
